armi/reactor/grids/tests/test_grids.py

```diff
@@ -353,17 +358,20 @@ def test_thirdAndFullSymmetry(self):
         self.assertEqual(third.getPositionsInRing(3), 12)
         self.assertEqual(third.getSymmetricEquivalents((3, -2)), [(-1, 3), (-2, -1)])
 
-    def test_pointsUpFlatsUp(self):
-        """Test the pointedEndUp attribute of the fromPitch method.
+    def test_cornersUpFlatsUp(self):
+        """Test the cornersUp attribute of the fromPitch method.
 
         .. test:: Build a points-up and a flats-up hexagonal grids.
             :id: T_ARMI_GRID_HEX_TYPE
             :tests: R_ARMI_GRID_HEX_TYPE
         """
-        tipsUp = grids.HexGrid.fromPitch(1.0, pointedEndUp=True)
-        flatsUp = grids.HexGrid.fromPitch(1.0, pointedEndUp=False)
+        cornersUp = grids.HexGrid.fromPitch(1.0, cornersUp=True)
+        self.assertAlmostEqual(cornersUp.pitch, math.sqrt(3) / 2)
```

Comment:
This unit test is a little funky. We initialize a grid from pitch by giving a pitch value of `1.0`, then assert that the pitch of the grid is equal to `math.sqrt(3) / 2`.

This is a bug that is fixed by #1654:

There are a few ways we could handle it:

1. Leave everything as is; both will get merged and the bug will be gone soon
2. We could just add that part of #1654 into this PR
3. Remove this part of the unit test and move it to #1654, where it can be implemented correctly

It feels weird to allow this unit test to be "wrong" in this commit and then fix it later, but this commit itself is not introducing a new bug, so I am okay with it. @john-science I would be fine with whichever of these paths you prefer.

Reply:
I added your bug fix in here:

https://github.com/terrapower/armi/pull/1649/commits/79a68cfa0f4ca4c348bfa920f5aef779f216d4e0

For the record, I didn't do that in the first place because you had opened a PR for it.  This is a fine solution to me.